CHANGELOG.mdx

```diff
@@ -16,6 +16,11 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [2.8.15] - 2021-11-25
+### Bugfixes
+- [#10381](https://github.com/rasahq/rasa/issues/10381): [#10381](https://github.com/RasaHQ/rasa/issues/10381): Validate regular
```

Comment:
💡 changelog fragments should not include links to issues. Otherwise it leads to issues like [this one](https://github.com/RasaHQ/rasa/issues/10452). The link to the issue is automatically added from the the name of the changlog fragment file. Please refer to [the documentation here](https://github.com/RasaHQ/rasa/blob/main/changelog/README.md), when in doubt 😄